com.github.alainm23.planner.json

```diff
@@ -0,0 +1,94 @@
+{
+  "app-id": "com.github.alainm23.planner",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
+  "sdk": "org.gnome.Sdk",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-19.08",
+  "command": "com.github.alainm23.planner",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--talk-name=ca.desrt.dconf",
+    "--talk-name=org.gtk.vfs.*",
+    "--talk-name=org.gnome.evolution.dataserver.AddressBook9",
+    "--talk-name=org.gnome.evolution.dataserver.Calendar7",
+    "--talk-name=org.gnome.evolution.dataserver.Sources5",
+    "--talk-name=org.gnome.evolution.dataserver.Subprocess.Backend.*",
+    "--filesystem=home",
+    "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
Any reason for re-adding dconf permissions?

Reply:
I think I need this to be able to use my variables in dconf.

Reply:
No, you don't need that anymore: https://blogs.gnome.org/mclasen/2019/07/12/settings-in-a-sandbox-world/.

Reply:
I am adding this line of code but it still does not work.
`"--metadata=X-DConf=migrate-path=/com/github/alainm23/planner/"`